src/shared/lib/comparison/ComparisonStore.ts

```diff
@@ -203,9 +203,11 @@ export default class ComparisonStore {
             };
             if (this.isLoggedIn) {
                 // need to add all groups belonging to this user for this origin
-                ret.user = (await comparisonClient.getGroupsForStudies(
```

Comment:
[Better formatting for inline await expression nested in calls](https://prettier.io/blog/2019/11/09/1.19.0.html#better-formatting-for-inline-await-expression-nested-in-calls-6856-by-thorn0)